### PR TITLE
Just some smal CS fixes for 14408

### PR DESCRIPTION
**Pull request by zero-24**
https://github.com/joomla/joomla-cms/pull/14408

### CODE REVIEW

File: administrator/components/com_users/views/users/tmpl/default.php [hunk 1]
```diff
@@ -16,9 +16,6 @@
 $listOrder  = $this->escape($this->state->get('list.ordering'));
 $listDirn   = $this->escape($this->state->get('list.direction'));
 $loggeduser = JFactory::getUser();
-
-
-// Create shortcut to parameters.
```
Comment (brianteeman) [inline administrator/components/com_users/views/users/tmpl/default.php:21]: Why remove this comment? You will find it elsewhere in the code base

**Review by brianteeman — COMMENTED** [review]
Ignore that comment. I shouldn't look in my phone. Will review tomorrow

**Comment by brianteeman** [comment]
thanks - merged it - looks fine now i can see it on my laptop

**Comment by zero-24** [comment]
Thanks Brian :+1:

**Pull request merged**